lsp-mode.el

```diff
@@ -6096,7 +6096,10 @@ Request codeAction/resolve for more info if server supports."
 
 (defun lsp-format-region (s e)
   "Ask the server to format the region, or if none is selected, the current line."
-  (interactive "r")
+  (interactive
+   (if (use-region-p)
+       (list (region-beginning) (region-end))
+     (list (point) (point))))
```

Comment:
Could it be `(list (line-beginning-position) (line-end-position))`? 🤔

Reply:
Current `(list (point) (point))` will be minimum changes for variant lsp backends; 
The `(list (line-beginning-position) (line-end-position))` maybe lead unexpected behavior on very long line for some lsp backends.